packages/internal/src/generate/plugins/rw-typescript-resolvers/visitor.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  TypeScriptResolversPluginConfig,
+  TypeScriptResolversVisitor,
+} from '@graphql-codegen/typescript-resolvers'
+import { FieldDefinitionNode, GraphQLSchema } from 'graphql'
+
+export class RwTypeScriptResolversVisitor extends TypeScriptResolversVisitor {
+  constructor(
+    pluginConfig: TypeScriptResolversPluginConfig,
+    schema: GraphQLSchema
+  ) {
+    super(pluginConfig, schema)
+  }
+
+  FieldDefinition(
+    node: FieldDefinitionNode,
+    key: string | number,
+    parent: any
+  ): (parentName: string) => string | null {
+    const hasArguments = node.arguments && node.arguments.length > 0
+
+    const superFieldDefinition = super.FieldDefinition(node, key, parent)
```

Comment:
Reusing all the logic from the original visitor implementation by calling the implementation in the super class